plugins/authsaml.php

```diff
@@ -1,13 +1,18 @@
 <?php
+require_once(__DIR__  . '/../vendor/autoload.php');
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__ . '/..');
```

Comment:
No need for the dotenv. Instead bundle the simplesamlphp code in plugins/authsaml/

Also, don' t put code here, it breaks phpList when loaded. Instead put it in the constructor.